.github/workflows/kubeapps-main-reusable.yaml

```diff
@@ -0,0 +1,754 @@
+# Copyright 2022 the Kubeapps contributors
+# SPDX-License-Identifier: Apache-2.0
+
+on:
+  workflow_call:
+    inputs:
+      run_gke_tests:
+        type: boolean
+        required: false
+        default: false
+
+env:
+  CHARTMUSEUM_VERSION: "3.9.1"
+  CHARTS_REPO_ORIGINAL: "bitnami/charts"
+  BRANCH_CHARTS_REPO_ORIGINAL: "main"
+  CHARTS_REPO_FORKED: "kubeapps-bot/charts"
+  BRANCH_CHARTS_REPO_FORKED: "main"
+  CI_BOT_USERNAME: "kubeapps-bot"
+  CI_BOT_EMAIL: "tanzu-kubeapps-team@vmware.com"
+  CI_BOT_GPG: "80B6EB16B1328FB18DFF2A073EBA68F3347E319D"
+  # DEBUG_MODE allows to activate some SSH debugging steps, and modify the verbosity level of some scripts (eg. e2e-tests.sh)
+  DEBUG_MODE: "true"
+  # FIXME: DEV_MODE is used to enable dev mode, used to activate several hacks along the code:
```

Comment:
Use the same convention for TODOs, this way, it easier to look for them in the future